### DISCUSSION TITLE
Super annoying build error

### DISCUSSION THREAD

**Discussion by FleetAdmiralJakob** [serwist/serwist#108] [Q&A]
I can't build my project because of this error:
`./public/sw.js:1:2
Type error: An expression of type 'void' cannot be tested for truthiness.`
https://pastebin.com/n7Xf3yY5

NOTE: I'm on the latest preview release

**Comment by DuCanhGH** [comment]
@FleetAdmiralJakob you don't usually lint/typecheck a bundled JavaScript file. Try adding "public/sw.js" to ".eslintignore" & "tsconfig.json#exclude".
